### DISCUSSION TITLE
I Use the IDialogAware in the UseControl, How do I not display the toolbar Like Window

### DISCUSSION THREAD

**Discussion by jietester** [PrismLibrary/Prism#2493] [WPF] [answered]
The `UseControl`  doesn't have the `WindowStyle= "None"` attribute 

![image](https://user-images.githubusercontent.com/25658690/123028148-8a9cda80-d411-11eb-8509-ce8ce37bc05c.png)

![image](https://user-images.githubusercontent.com/25658690/123028152-8e306180-d411-11eb-99eb-6343e77454de.png)

![image](https://user-images.githubusercontent.com/25658690/123028130-82dd3600-d411-11eb-9e6d-196be2ef5a9b.png)

**Comment by LGTCO** [comment]
you should rewrite DialogWindow's Style by yourself.

**Reply by jietester** [reply]
Thinks.

**Comment by brianlagunas** [comment, marked as answer]
Read these docs https://prismlibrary.com/docs/wpf/dialog-service.html#style-the-dialogwindow

**Reply by jietester** [reply]
Thinks,  I see.
